system/templates/rolls/roll-graphic.hbs

```diff
@@ -0,0 +1,54 @@
+<div class="dice-roll ironsworn-roll">
+
+  <div class="dice-formula">
+    <p>
+      {{!-- The action die or progress score --}}
+      <span class="action roll d6 {{actionMinMax}}
+        {{#if actionCanceledByNegativeMomentum}}canceled{{/if}}
+      " title="{{computedActionDie.source}}">
+        {{#if computedActionDie.value}}
+        {{computedActionDie.value}}
+        {{else}}
+        &nbsp;
+        {{/if}}
+      </span>
+
+      {{#each adds}}
+      &plus;
+      <span title="{{source}}">{{value}}</span>
+      {{/each}}
+
+      {{#if actionTotal}}
+      =
+      <strong>
```

Comment:
`<strong>` may or may not be the best choice here, depending on what you want to express -- it's got some pretty assertive semantics attached to it. longer discussion of it here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong

Reply:
This whole display will probably get reworked to look more like the example in #366, so let's table this for now.